### PR TITLE
Feat (relay): Added relay list

### PR DESCRIPTION
**Pull request by priscilawebdev**
Preview:

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx [hunk 1]
```diff
@@ -1,23 +1,190 @@
...
 type Props = {
   organization: Organization;
 } & RouteComponentProps<{orgId: string}, {}>;
 
-class Relays extends AsyncComponent<Props> {
+type State = AsyncComponent['state'] & {
+  relays: Array<Relay>;
+};
+
+const relaysMock: Array<Relay> = [
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:38]: Is this view hidden behind a feature flag?

File: src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx [hunk 2]
```diff
@@ -1,23 +1,190 @@
...
+                      <Time label={t('First Seen:')} date={first_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last Seen:')} date={last_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last modified:')} date={last_modified} />
+                    </InfoItem>
+                  </Info>
+                  <Tooltip title={t('Delete Rule')}>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:115]: You can use the `title` attribute on a button to add a tooltip with fewer lines of code.

File: src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx [hunk 3]
```diff
@@ -1,23 +1,190 @@
...
+`;
+
+const Info = styled('div')`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: ${space(1)};
+  > *:nth-child(1),
+  > *:nth-child(2) {
+    grid-column: span 4;
+  }
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:161]: What does this do?

File: src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:38]: yes :)

File: src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx (hunk 3 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:161]: the first and the second grid-cell will take 4 columns. I could also use grid-column-start: 1 and grid-column-end: -1

Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx:159]: Oh I see -- I didn't realize Name/PubKey was part of the grid.

**Review by billyvg — APPROVED** [review]
lgtm!

**Pull request merged**